docs/source/release/v6.11.0/mantidworkbench.rst

```diff
@@ -7,13 +7,13 @@ Mantid Workbench Changes
 
 New Features
 ------------
+- There is a new :ref:`Deprecation Policy <deprecation_policy>` for Mantid.
```

Comment:
This note should be with the framework release notes

Reply:
done